jobs/worker/spec

```diff
@@ -178,6 +178,13 @@ properties:
 
       If specified, the other garden.* properties have no effect.
 
+  garden.request_timeout:
+    env: CONCOURSE_GARDEN_REQUEST_TIMEOUT
+    description: |
+      How long to wait for requests to Garden to complete.
+      0 means no timeout.
+    default: 5m
```

Comment:
ditto with the [default](https://github.com/concourse/concourse/blob/master/worker/workercmd/worker_linux.go#L38).
```suggestion
```